decidim-core/lib/decidim/core.rb

```diff
@@ -149,6 +149,11 @@ def self.seed!
     true
   end
 
+  # Setting that
```

Comment:
Seems like the comment explaining what's doing is missing :/

Reply:
@andreslucena the documentation has been added.